Add available data to storage and hrmem

arihantbFirstwave opened the pull request.

arihantbFirstwave committed:
Add available data to storage and hrmem

arihantbFirstwave committed:
add exception handling in Host_resource plugin

markdueck1 commented on cgi-bin/network.pl:
```diff
@@ -3713,6 +3730,12 @@ sub viewStorage
+			if (defined $D->{hrStorageAvailUnits}){
+				$available_data = $D->{hrStorageUnits} * $D->{hrStorageAvailUnits};
+			}else{
+				$available_data = $free + $cached_data + $buffers_data;
```
we do not want this calculation in 3 places, if the value is not present just don't show it

markdueck1 commented on lib/NMISNG/Node.pm:
```diff
@@ -7615,6 +7616,11 @@ sub collect_server_data
 						$S->{reach}{memused} = $D->{hrMemUsed}{value};
+						# calculate available memory
+						if ($storage_target->{hrStorageDescr} eq 'Physical memory'){
+							$D->{hrMemAvail}{value} = $S->{reach}{memfree} + $cached_units + $buffer_units;
```
This needs error checking for hash or 0  like the other section

Commit:
Merge branch 'nmis9_dev' into feature/OMK-11038

Commit:
Available memory adjustments

Keep available memory value with units in, do not divide it out, that causes the values to differ too much when putting them back together

change the graphs so they are organised a little more and have lesss duplicate info in the legend
